legend-engine-xts-graphQL/legend-engine-xt-graphQL-pure/src/main/resources/core_external_query_graphql/transformation/dynamic/expressions/and.pure

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+###Pure
+import meta::external::query::graphQL::transformation::queryToPure::dynamic::utility::*;
+import meta::external::query::graphQL::transformation::queryToPure::dynamic::expressions::and::*;
+import meta::external::query::graphQL::metamodel::sdl::*;
+import meta::external::query::graphQL::metamodel::sdl::value::*;
+import meta::external::query::graphQL::metamodel::sdl::executable::*;
+import meta::external::query::graphQL::transformation::queryToPure::dynamic::*;
+import meta::external::query::graphQL::transformation::queryToPure::dynamic::metamodel::*;
+import meta::pure::graphFetch::*;
+import meta::pure::functions::lang::*;
+import meta::pure::functions::multiplicity::*;
+
+Class meta::external::query::graphQL::transformation::queryToPure::dynamic::metamodel::AndExpression extends meta::external::query::graphQL::transformation::queryToPure::dynamic::metamodel::BooleanExpression
+{
+  first: meta::external::query::graphQL::transformation::queryToPure::dynamic::metamodel::BooleanExpression[1];
+  second: meta::external::query::graphQL::transformation::queryToPure::dynamic::metamodel::BooleanExpression[1];
+}
+
+function meta::external::query::graphQL::transformation::queryToPure::dynamic::expressions::and::processAnd(o: ObjectField[1], prefix: String[1], debug: DebugContext[1]): Node[*]
```

Comment:
I think we need better names - I see two functions - processAnd and processAndExpression

Reply:
noted